Updated and synced feature lists and updated redirect links

UtiluMark opened the pull request:
Updated and synced the feature lists in the README and the About window.
Updated the AMO links in the README and the About window to prevent
redirects.
Added link to Screengrab (fix version) add-on.
Part of #188

whimboo commented on History.md:
```diff
@@ -58,6 +58,6 @@
   * Extension: Update list of contributors (#38)
   * Extension: Update maxVersions for applications (#37)
-  * Extension: Allow title bar customization in Thunderbird (#24)
+  * Extension: Allow titlebar customization in Thunderbird (#24)
```
The former wording was correct. So please revert this change.

UtiluMark replied:
@whimboo Do you want all the occurrences of "titlebar" changed to "title bar"? Also in the locales?

> History.md
> Line 14: * Extension: Make dtd entities for Customize Titlebar more localization friendly. (#140)
> Line 16: * Extension: Add basic Seamonkey support to titlebar customization. (#31)
> Line 34: * Extension: Make columns of 'Customize Titlebar' dialog's tree resizable. (#25)
> Line 41: * Extension: Open customize.xul (Customize Titlebar) as a resizable window. (#55)
> Line 43: * Extension: Include changeset info in the Insert/Copy Build ID menuitem and in 'Customize Titlebar'. (#65)
> Line 61: * Extension: Allow titlebar customization in Thunderbird (#24)
> Line 62: * Extension: Version 3.1 loses ability to customize titlebar with just window title (#18)

> \extension\chrome\locale\en-US\customize.dtd
> Line 10: <!ENTITY nightly.customize.title              "Customize Titlebar">
> Line 12: <!ENTITY nightly.paneTitlebar.title           "Titlebar">
> Line 13: <!ENTITY nightly.paneTitlebar.description     "Titlebar Customization">

> \extension\chrome\locale\en-US\nightly.dtd
> Line 23: <!ENTITY nightly.customize.label              "Customize Titlebar">

whimboo replied:
We don't want to change the entity names but it might be good to update the actual output. @xabolcs what do you think?

xabolcs replied:
Sure, keep entity names.

UtiluMark replied:
@whimboo @xabolcs Shall I replace "titlebar" with "title bar" everywhere except in the entity names?

whimboo replied:
I would say yes, at least for en-US. Maybe we should not touch other locales.

whimboo commented on README.md:
```diff
@@ -1,12 +1,15 @@
 * Open Profile Folder
+* Open Pushlog
+* Screenshots utility
+* Titlebar customization
```
Lets fix `Title bar` here too.

whimboo commented on extension/chrome/content/aboutNightly/aboutNightly.xhtml:
```diff
@@ -44,11 +44,15 @@
+      <li>Open Profile Folder</li>
+      <li>Open Pushlog</li>
+      <li>Screenshots utility</li>
+      <li>Titlebar customization</li>
```
and here.

whimboo commented:
Only some nits left here. Otherwise it looks fine to me. Thanks.

UtiluMark commented:
@whimboo I've replaced "titlebar" with "title bar" everywhere except in entity names and reverted the Copy about:support to Clipboard feature to Pastebin.
Please review and merge.

whimboo commented on README.md:
```diff
@@ -37,4 +40,5 @@ All bugs are reported to the Nightly Tester Tools component at bugzilla.mozilla.
 # Related Add-ons:
-* [Add-on Compatibility Reporter](https://addons.mozilla.org/en-US/firefox/addon/15003/)
+* [Screengrab (fix version)](https://addons.mozilla.org/en-US/firefox/addon/screengrab-fix-version/)
+* [Add-on Compatibility Reporter](https://addons.mozilla.org/en-US/firefox/addon/add-on-compatibility-reporter/)
```
Lets keep the add-on compat reporter as first item so we have an alphabetical sorted list.

UtiluMark replied:
Done.

whimboo commented:
Looks fine to me beside the nit. Once an update has been pushed and you (@xabolcs) are ok with the changes, can you please get it landed? Thanks.

xabolcs commented:
> Once an update has been pushed and you (@xabolcs) are ok with the changes, can you please get it landed?

Sure!

UtiluMark commented:
@xabolcs Update done, so you can get this landed.

xabolcs commented:
@UtiluMark 
Sorry for last minute review, but what are the benefits preventing redirects of AMO links?

whimboo commented:
FYI by default we squash all commits at the end and give the final commit a proper comment (please see git log). @UtiluMark maybe you can do it to make it easier for @xabolcs to get it landed.

UtiluMark commented:
@xabolcs The benefits of preventing redirects of AMO links are that the current links have more readable URLs (for example "nightly-tester-tools" in the URL is more descriptive compared to the number "6543" of the old URL).
The old AMO links are currently redirected to the new ones using a 301 MOVED PERMANENTLY redirect, these redirects might get removed at some point in the future since the URLs changed permanently, so the old AMO links might end up in 404 FILE NOT FOUND errors in the future.

xabolcs commented:
Thanks @UtiluMark!

xabolcs commented:
Looks good to me.

Ohh, almost forgot. @whimboo could you provide a commit message for this PR? :)
Thanks!

whimboo commented:
I would like to let @UtiluMark do the squashing of commits and the commit message as pointed out in my last comment. @UtiluMark do you think you can do it? If you need help with let us know. In general make use of `git rebase -i master` and fixup all commits except the first one which needs an updated message.

xabolcs commented:
@whimboo it's OK to me if you would like to teach @UtiluMark contributing to NTT, but a squashed commit of this PR is already in my working copy. It's OK to me to do the squashing work. :)

UtiluMark commented:
@xabolcs I've just done what @whimboo asked. If you already have a squashed commit thats fine. :)

xabolcs commented:
@UtiluMark, a `git commit --amend` is missing for me, to update the commit message in my copy.

UtiluMark commented:
@xabolcs Done, changed the commit message to "Updated and synced feature lists and updated redirect links".

whimboo commented:
There are still 39 commits listed on this PR. You might not have pushed via -f. Also for comments we add the issue number in brackets at the end.

UtiluMark commented:
So the commit message for this PR should be:
> Updated and synced feature lists and updated redirect links (#198)
@xabolcs You wrote you already have a squashed commit of this PR, so there is nothing more to be done for me, right?

xabolcs commented:
@UtiluMark, sure, but @whimboo would like to let you do all the work. :)

The commit message is almost correct. It should refer to #188 for now.

xabolcs commented:
@UtiluMark, take the squash & rebase exercises on the next PR! :)

Merging this.

xabolcs commented:
Landed as commit f313b2b2f6fb950cb5ff6633897c91ceb47b6255.
